### ISSUE TITLE
Allegro One - zmiana nazw metod dostawy?

### ISSUE THREAD

**Issue by omnihero1** [allegro/allegro-api#8813]
Czy nazwy metod dostawy dla Allegro One zostały zmnienione 2 kwietnia?

![image](https://github.com/allegro/allegro-api/assets/134368877/b6e57000-a8f0-4a54-945b-a87cbe8f67cc)

To co widać od razu to to, że metody z „UPS” po przecinku wypierają działające do tej pory metody Allegro One Box i Allegro One Punkt. Patrząc na ilości zamówień:

Nasza teoria jest taka, że dotychczas nie było rozróżnienia dla Allegro One z UPS w nazwie, bo tylko UPS obsługiwał te metody dostawy. W momencie jak rozpoczęliście przygotowania do dodania DPD do tego serwisu, aby rozróżnić przewoźników dodaliście do dotychczasowych nazw prefiksy kurierów jak „UPS” (widzimy te błędy od 2 kwietnia, czyli na 2 tygodnie przed dodaniem DPD, który zapewne już mocno testujecie).

Czy nasza teoria jest poprwna? Mamy w integracji problem od 2 kwietnia z obsługa zamówień dla One Boxa

**Comment by MaciejFrackowiak** [comment]
Tak, dodane zostały przedrostki konkretnego przewoźnika. Co istotne i kluczowe - identyfikatory metod dostawy pozostały te same i to na nich należy opierać integrację, a nie na nazwach, które mogą być zmieniane w zależności od zmian otoczenia biznesowego.

**Comment by omnihero1** [comment]
a gdzie znajdę identyfikatory? z tego co widze to są tylko w "starym api" ktore jest już "deprecated"

 Ponieważ korzystamy z najnowszego API Allegro (https://developer.allegro.pl/documentation/#operative/getListOfOrdersUsingGET), nie jestem w stanie uzyskać tych informacji, chyba że patrze na zły endpoint.

![image](https://github.com/allegro/allegro-api/assets/134368877/10b6d04e-b728-4b51-b596-28a7a797d0c7)
